index.html

```diff
@@ -0,0 +1,161 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/6b55e5dec8.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="stylesheet" href="//use.fontawesome.com/releases/v5.0.7/css/all.css">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="about.css">
+    <title>HIV Global Summit 2022</title>
+</head>
+<body id="body">
+<nav class="navigation-bar">
+    <a href="/" class="social__icons--link" target="_blank">
+        <i class="fab fa-twitter"></i>
+    </a>
+    <a href="/" class="social__icons--link" target="_blank">
+        <i class="fab fa-facebook"></i>
+    </a>
+    <a href="/" class="social__icons--link" target="_blank">
+        Kampala
+    </a>
+    <a href="/" class="social__icons--link" target="_blank">
+        Uganda
+    </a>
+</nav>
+<header>
+    <nav class="desktop-nav">
+        <div class="logo-image">
+            <img src="assets/images/download.png" alt="">
+            <a href="/" class="logo-home">CC HIV Global Summit 2022</a>
+        </div>
+        <div>
```

Comment:
- [ ] The logo and the home menu options are broken, When I click on them I am supposed to redirect to the homepage but it's not. Please fix this issue.

### See the image for reference 

![image](https://user-images.githubusercontent.com/83803414/169515614-4ebcc491-5a19-4307-b468-e7d81b073090.png)